src/chunk_list.h

```diff
@@ -797,6 +798,16 @@ static_inline bool chunk_is_forin(chunk_t *pc)
 }
 
 
+/**
+ * Checks if a chunk is a keyword
+ *
+ * @return true  - the chunk is keyword
+ */
+static_inline bool chunk_is_keyword(chunk_t *pc)
+{
+   return(find_keyword_type(pc->text(), strlen(pc->text())) != CT_WORD);
```

Comment:
Small stylistic comment. You shouldn't wrap it in parens unless it has semantic value. Which it doesn't do in this case.

When quickly scanning the code, you would think that return is a function.

Reply:
Unfortunately it's the style chosen and enforced by upstream.

Reply:
Hmm ... ok. Makes me a bit worried that it actually adds the parens. But if this is the style for the uncrustify repo, then thats how it is.